README.md

```diff
@@ -2,18 +2,50 @@
 
 [![CircleCI](https://circleci.com/gh/okteto/cnd.svg?style=svg)](https://circleci.com/gh/okteto/cnd)
 
-**Cloud Native Development** (CND) is about running your development flow entirely in kubernetes, avoiding the time-consuming `docker build/push/pull/redeploy` cycle.
+**Cloud Native Development** (CND) is about moving your entire development workflow to kubernetes, avoiding the time-consuming `docker build/push/pull/redeploy` cycle.
 
 CND helps you achieve this with a mix of kubernetes automation, file synching between your local file system and kubernetes and hot reloading of containers.
 
 ## Quickstart
 
-- Install `cnd` locally 
-- Deploy a kubernetes service
-- Run `cnd up`
-- 
+Let's start with something simple to show you the power of cloud native development. First, install [cnd locally](#installation).  
 
-For a more advanced scenario [check out our voting app cnd demo](https://github.com/okteto/cnd-voting-demo).
+Clone the `welcome` service.
```

Comment:
I would move the example here. It is a common practice in other dev tools

Reply:
My thought here is that it's kind of odd to clone the repo of a tool just to get a sample of how to use it, so instead we can have a samples repo (maybe put vote the getting started and the more advance one there?) (telepresence and duffle do this)